gocd/test.json

```diff
@@ -0,0 +1,257 @@
+{
+  "_links": {
+    "self": {
+      "href": "http://10.95.74.117:8153/go/api/admin/pipelines/test"
+    },
+    "doc": {
+      "href": "http://api.go.cd/#pipeline_config"
+    },
+    "find": {
+      "href": "http://10.95.74.117:8153/go/api/admin/pipelines/:name"
+    }
+  },
+  "label_template": "${COUNT}",
+  "enable_pipeline_locking": false,
+  "name": "test",
+  "template": null,
+  "parameters": [
+
+  ],
+  "environment_variables": [
+    {
+      "secure": false,
+      "name": "PGHOST",
+      "value": "PGHOST"
+    },
+    {
+      "secure": false,
+      "name": "PGPORT",
+      "value": "5432"
+    },
+    {
+      "secure": false,
+      "name": "PGDATABASE",
+      "value": "test_development"
+    },
+    {
+      "secure": false,
+      "name": "PGUSER",
+      "value": "PGUSER"
+    },
+    {
+      "secure": false,
+      "name": "PGPASSWORD",
+      "value": "PGPASSWORD"
+    },
+    {
+      "secure": false,
+      "name": "PGSSLMODE",
+      "value": "require"
+    }
+  ],
+  "materials": [
+    {
+      "type": "git",
+      "attributes": {
+        "url": "https://github.cfpb.gov/databases/test.git",
```

Comment:
+1 @frank0631 can you replace that with a public url?  Something like https://github.com/cfpb/rhobot-test.git?